docs/source/recipes/images.md

```diff
@@ -0,0 +1,66 @@
+---
+myst:
+  html_meta:
+    "description": "Volto code recipe for images"
+    "property=og:description": "Volto code recipe for images"
+    "property=og:title": "Volto code recipe for images"
+    "keywords": "Plone, Volto, code, recipe, images"
+---
+
+(images-label)=
+
+# Images
+
+Volto comes with an `Image` component that loads images in an optimal way.
+
+Features of the `Image` component are the following.
+
+- optimized [lazy loading](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/loading)
+- automatic `srcset` generation, if given a content item or catalog brain
+- optional responsive width
+
+The component can accept all the regular HTML `<img>` attributes and a few extra for the above features.
+
+Setting the `loading="lazy"` attribute to the `Image` component also adds the `decoding="async"` attribute.
+The [`async`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/decoding) attribute decodes the image asynchronously to reduce delay in presenting other content.
+
+To render an image from a content object, you can use the component as shown in the following code snippet.
+
+```jsx
+import config from '@plone/volto/registry';
+
+const Image = config.getComponent({ name: 'Image' }).component;
+
+<Image item={content} imageField="image" alt="" />;
```

Comment:
I just realized we should set a good example. Empty `alt` makes me sad. 😢 

```suggestion
<Image item={content} imageField="image" alt="Alt text for accessibility" />;
```

Reply:
Generally speaking, empty `alt`s are not necessarily bad :)
I agree we should emphasize that usage is important for a11y though, so we should add it here.

Reply:
Agreed, however I would assume that these are images used in content blocks that add meaning to the page, and are not purely decorative images. For posterity, this is a good guide to determine whether to use `alt`: https://www.w3.org/WAI/tutorials/images/decision-tree/